src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.dao;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    Optional<MovieSession> get(Long id);
+
+    List<MovieSession> getAll();
```

Comment:
there wasn't this method in our task - in CM first statements is: "Do not create redundant functionality, like methods that are not required by the task.")